.github/scripts/runner_determinator.py

```diff
@@ -1,16 +1,62 @@
-import json
+# flake8: noqa: G004
```

Comment:
It would be nice to leave a comment why one believes its OK to ignore this lint rule...